src/index.html

```diff
@@ -17,11 +20,378 @@
+      href="styles/index.css"
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#"
+      >
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--active is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--hover"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="grid-container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <div class="stars stars--active">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-number">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="buy"
```

Comment:
The 'data-qa' attribute is used here, but it is not clear if it is necessary for the functionality or testing. Ensure that this attribute is needed, or remove it if it is not used.

Reply:
Ignore this AI comment, data-qa is necessary